src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.html

```diff
@@ -3,13 +3,17 @@
 
         <umb-editor-header
             name="model.title"
+            navigation="vm.navigation"
+            on-select-navigation-item="vm.onNavigationChanged(item)"
```

Comment:
These are duplicated

Reply:
Funky, interesting that Patrick and I wrote almost identical code, resulting in merging resulted in this.